lib/__tests__/Autocomplete-test.js

```diff
@@ -41,12 +48,14 @@ describe('Autocomplete acceptance tests', () => {
 
     expect(autocompleteWrapper.state('isOpen')).to.be.false;
     expect(autocompleteWrapper.instance().refs.menu).to.not.exist;
+    expect(onFocusSpy).to.not.have.been.called()
```

Comment:
Once you get the spy harness sorted you shouldn't have to check their initial state every time.